src/Native/Time.js

```diff
@@ -12,37 +12,36 @@ Elm.Native.Time.make = function(elm) {
 
   function fpsWhen(desiredFPS, isOn) {
```

Comment:
Do you find the old version of this function less confusing? That would be https://github.com/elm-lang/core/blob/master/src/Native/Time.js#L13-39. It has the same basic structure with the non-exposed signals (though nothin for Elm Reactor to pick up).

The crucial commit concerning this function is this one: https://github.com/jvoigtlaender/core/commit/274cd7eb9a05c1b32dab563a871c557097bc3392. All subsequent commits are just renamings or moving around of variables, prompted by @laszlopandy's comments.

Should I talk you through the mentioned commit? I'd say it very much *simplifies* this function compared to how it was previously.